src/platforms/flutter/migration.mdx

```diff
@@ -3,6 +3,10 @@ title: Migration Guide
 sidebar_order: 1000
 ---
 
+## Migrating from `sentry_flutter` `5.1.x` to `sentry_flutter` `6.0.0`
```

Comment:
Can you please also fix the two titles after this one by changing "from" to "From" in the first one and changing the next one to "Android-Specific Changes". Also please make the Android heading an H2 so that it shows up in the right sidebar.

```suggestion
## Migrating From `sentry_flutter` `5.1.x` to `sentry_flutter` `6.0.0`
```